gotasks/specgen_task.go

```diff
@@ -43,6 +43,24 @@ var dcpMetadata = []DCPMetadata{
 		OfficialName: "Internet Gateway Device v1",
 		DocURL:       "http://upnp.org/specs/gw/UPnP-gw-InternetGatewayDevice-v1-Device.pdf",
 		XMLSpecURL:   "http://upnp.org/specs/gw/UPnP-gw-IGD-TestFiles-20010921.zip",
+		Hacks: []DCPHackFn{
+			func(dcp *DCP) error {
+				for _, service := range dcp.Services {
+					scpd := dcp.Services[3].SCPD
```

Comment:
Like the variable is searched for by name, search for the correct service by an identifier, it'll be less brittle. Using the URN field to identify should work nicely.